The first hunk fixes the build by including the right FreeType directory.

The second one explicitly links against libkdeui, as there were linking
problems because of kdeui functions being used without the library being
explicitly referenced.
--- CMakeLists.txt.orig	2009-08-31 18:17:15.000000000 +0300
+++ CMakeLists.txt	2013-08-11 19:53:43.000000000 +0300
@@ -11,7 +11,7 @@
 
 configure_file(config-kpovmodeler.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-kpovmodeler.h )
 
-include_directories(${FREETYPE_INCLUDE_DIR})
+include_directories(${FREETYPE_INCLUDE_DIRS})
 add_definitions (-DQT3_SUPPORT -DQT3_SUPPORT_WARNINGS)
 
 ########### next target ###############
@@ -306,7 +306,7 @@
 
 kde4_add_executable(kpovmodeler ${kpovmodeler_SRCS})
 
-target_link_libraries(kpovmodeler  ${KDE4_KDECORE_LIBS} lkpovmodeler )
+target_link_libraries(kpovmodeler ${KDE4_KDECORE_LIBS} ${KDE4_KDEUI_LIBS} lkpovmodeler )
 
 install(TARGETS kpovmodeler  ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
